src/test/java/org/prebid/server/bidder/invibes/InvibesBidderTest.java

```diff
@@ -0,0 +1,242 @@
+
+    private static final String ENDPOINT_URL = "https://{{Host}}/test";
+
+    private InvibesBidder invibesBidder;
+
+    @Before
+    public void setUp() {
+        invibesBidder = new InvibesBidder(ENDPOINT_URL, jacksonMapper);
+    }
+
+    @Test
+    public void creationShouldFailOnInvalidEndpointUrl() {
+        Assertions.assertThatIllegalArgumentException().isThrownBy(() ->
+                new InvibesBidder("invalid_url", jacksonMapper));
+    }
+
+    @Test
+    public void makeHttpRequestsShouldCreateCorrectURL() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(identity(), ExtImpInvibes
+                .of("12", 1003, InvibesDebug.of("test", true)))
+                .toBuilder().site(Site.builder().page("www.test.com").build()).build();
+
+        // when
+        final Result<List<HttpRequest<InvibesBidRequest>>> result = invibesBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1);
+        assertThat(result.getValue().get(0).getUri()).isEqualTo("https://bid3.videostep.com/test");
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorWhenImpExtCouldNotBeParsed() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder()
+                        .ext(mapper.valueToTree(ExtPrebid.of(null, mapper.createArrayNode())))
+                        .build()))
+                .build();
+
+        // when
+        final Result<List<HttpRequest<InvibesBidRequest>>> result = invibesBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).hasSize(1);
+        assertThat(result.getErrors().get(0).getMessage()).startsWith("Error parsing invibesExt parameters");
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorWhenBannerIsNull() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(identity(), ExtImpInvibes
+                .of("12", 15, InvibesDebug.of("test", true)));
+
+        final BidRequest bidRequestWithoutBanner = bidRequest.toBuilder().imp(
+                Collections.singletonList(bidRequest.getImp().get(0).toBuilder().banner(null).build())).build();
+
+        // when
+        final Result<List<HttpRequest<InvibesBidRequest>>> result =
+                invibesBidder.makeHttpRequests(bidRequestWithoutBanner);
+
+        // then
+        assertThat(result.getErrors()).hasSize(1)
+                .containsOnly(BidderError.badInput("Banner not specified"));
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorWhenSiteIsNotPresent() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(identity(), ExtImpInvibes
+                .of("12", 15, InvibesDebug.of("test", true)));
```

Comment:
You also can create simple givenBidRequest method which will return bidRequest with this parameters. (this is third similar call) (first method also technicality can call this parameter)